e2e/client/playwright/templates.spec.ts

```diff
@@ -0,0 +1,113 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {restoreDatabaseSnapshot, s} from './utils';
+
+test('creating new template', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/templates');
+
+    await page.locator(s('template-header')).getByRole('button', {name: 'Add new'}).click();
+
+    await page.locator(s('template-edit-view')).getByPlaceholder('template name').fill('Template 1');
+    await page.locator(s('template-edit-view', 'select-content-profile')).selectOption({label: 'Story'});
+    await page.locator(s('template-edit-view')).getByRole('button', {name: 'Save'}).click();
+
+    await expect(page.locator(s('template-content', 'content-template--template 1'))).toBeVisible();
+});
+
+test('editing template', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/templates');
+
+    await page.locator(s('template-content', 'content-template--story', 'template-actions')).click();
+    await page.locator(s('template-actions--options')).getByRole('button', {name: 'Edit'}).click();
```

Comment:
When you are testing something that has dynamic test IDs, take your time and convert it to data-test-id/data-test-value. Especially when it's easy to do so - there are only 2 lines in old tests that you'd need to update.